src/hooks/Auth/useAuthn.ts

```diff
@@ -40,7 +40,7 @@ export default (): {
   }
 
   const getAuthnList = async (isInit = false) => {
-    isListLoading.value = !isInit && true
+    !isInit ? (isListLoading.value = true) : void 0
```

Comment:
Hmmm... You should actually just use an if statement? 😳

```suggestion
   if (!isInit) isListLoading.value = true
```